src/main/java/net/atos/zac/app/informatieobjecten/model/validation/ValidRestEnkelvoudigInformatieFileUploadForm.java

```diff
@@ -16,7 +16,7 @@
 @Constraint(validatedBy = {ValidRestEnkelvoudigInformatieFileUploadFormValidator.class})
 public @interface ValidRestEnkelvoudigInformatieFileUploadForm {
 
-    String INVALID_FILE_UPLOAD_FORM = "Uploaded file is empty";
+    String INVALID_FILE_UPLOAD_FORM = "Een leeg bestand is niet toegestaan";
```

Comment:
So we don't translate these error messages in the frontend? We should really do that or preferably even start working with error codes even (but that's for later of course).

Reply:
@felixcicatt Do you know if we translate messages? Should the above be part of a i18n bundle?